lib/terraforming/resource/route53_record.rb

```diff
@@ -62,7 +63,9 @@ def record_id_of(record, zone_id)
       def record_sets_of(hosted_zone)
         @client.list_resource_record_sets(hosted_zone_id: zone_id_of(hosted_zone)).map do |response|
           response.data.resource_record_sets
-        end.flatten
+        end.flatten.select do |resource|
```

Comment:
__[rubocop]__
* Avoid multi-line chains of blocks. [:ref](https://github.com/bbatsov/ruby-style-guide#single-line-blocks)

<!-- Comment by [SideCI](https://sideci.com) -->